modules/op-installing-and-deploying-tekton-hub-on-an-openshift-cluster.adoc

```diff
@@ -0,0 +1,105 @@
+// This module is included in the following assembly:
+//
+// *cicd/pipelines/using-tekton-hub-with-openshift-pipelines.adoc
+
+:_content-type: PROCEDURE
+[id="op-installing-and-deploying-tekton-hub-on-an-openshift-cluster_{context}"]
+= Installing and deploying {tekton-hub} on an OpenShift cluster
+
+[role="_abstract"]
+{tekton-hub} is an optional component; cluster administrators cannot install it using the `TektonConfig` custom resource (CR). To install and manage {tekton-hub}, use the `TektonHub` CR.
+
+[NOTE]
+====
+If you are using Github Enterprise or Gitlab Enterprise, install and deploy {tekton-hub} in the same network where the enterprise server is running. For example, if enterprise server is running behind VPN, then the Openshift cluster on which you are deploying {tekton-hub} must also be behind VPN.
```

Comment:
```suggestion
If you are using Github Enterprise or Gitlab Enterprise, install and deploy {tekton-hub} in the same network where the enterprise server is running. For example, if the enterprise server is running behind a VPN, deploy {tekton-hub} on a Openshift cluster that is behind the VPN.
```

Reply:
If you refer to the OpenShift product, use the {product-title} attribute.

Reply:
This line caught my eye because of the missing articles in "if the enterprise server is running behind a VPN,". I've suggested some further simplifications of the last phrase.

Reply:
s/Gitlab/GitLab/